include/swift/AST/DiagnosticsSema.def

```diff
@@ -88,6 +88,14 @@ ERROR(could_not_find_subscript_member_did_you_mean,none,
       "did you mean to use the subscript operator?",
       (Type))
 
+ERROR(could_not_find_subscript_member_tuple, none,
+      "tuple type %0 element cannot be accessed using subscript",
+      (Type))
+ERROR(could_not_find_subscript_member_tuple_did_you_mean_use_dot, none,
+      "tuple type %0 element cannot be accessed using subscript; "
+      "did you mean to use '.' to access element?",
+      (Type))
```

Comment:
This doesn't quite sound right. It feels like there is a missing `'s` somewhere, but "tuple type %0's element" also doesn't sound right. What do you think of the following alternative:

1. For the non-fix-it case: "cannot access element using subscript for tuple type '%0'; use '.' notation instead".
2. For the fix-it case: "cannot access element using subscript for tuple type '%0'; did you mean to use '.%1'?"

Reply:
Oh right adjusted :)

Reply:
Also added labeled tuple test cases